utils/segprint.py

```diff
@@ -20,7 +22,6 @@ class NoDB(Exception):
 # int and sites are loaded together so that bit coverage can be checked together
 # however, as currently written, each segment is essentially printed twice
 def process_db(db, tile_type, process, verbose):
-    print(db.get_tile_types())
     ttdb = db.get_tile_type(tile_type)
 
     fns = [ttdb.tile_dbs.segbits, ttdb.tile_dbs.ppips]
```

Comment:
Could use TileSegbits via db.get_tile_segbits? 
https://github.com/SymbiFlow/prjxray/blob/master/prjxray/db.py#L138

Reply:
Sounds good to me.